enclave/src/c_api/c_api.cc

```diff
@@ -430,6 +538,11 @@ int XGDMatrixCreateFromEncryptedFile(const char *fnames[],
         oss << " username " << usernames[i] << " filename " << fnames[i];
     }
     oss << " silent " << silent;
+    oss << " nonce ";
```

Comment:
This code is repeated in many functions. Can we write a helper function for this?